components/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
-# [0.0.12] - 2022-01-27
+# [0.0.13] - 2022-01-31
```

Comment:
Version can be removed - date is enough as version is provided on GitHub releases page.